test/e2e/util.go

```diff
@@ -30,14 +31,19 @@ import (
 )
 
 const (
-	vcsim        = "vcsim"
-	ns           = "default"
-	vsphereCreds = "vsphere-credentials"
-	user         = "user"
-	password     = "password"
-	jobNameKey   = "job-name"
+	vcsim             = "vcsim"
+	ns                = "default"
+	vsphereCreds      = "vsphere-credentials"
+	user              = "user"
+	password          = "password"
+	jobNameKey        = "job-name"
+	DefaultVcsimImage = "vmware/vcsim:latest"
```

Comment:
nit: needs to be exported?

Reply:
(Actually, it looks like this isn't used anymore with the use of a default on the `envConfig` struct.)

Reply:
used in a comparison in line 455

Reply:
yeah, I would've liked to only specify the image name in one place, but I can't use variables in struct tags afaik.